docs/api/openapi-worklog.md

```
@@ -0,0 +1,125 @@
+3. `POST /v1/auth/register` - done
+4. `POST /v1/auth/session` - done
+5. `GET /v1/auth/profile` - done
+6. `PATCH /v1/auth/profile` - done
+7. `GET /v1/jobs` - done
+8. `POST /v1/jobs` - done
+9. `GET /v1/jobs/{id}` - done
+10. `PUT /v1/jobs/{id}` - done
+
+## Chunk 2 - done
+
+11. `DELETE /v1/jobs/{id}` - done
```

markdownlint-cli2 (0.21.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 11; Style: 1/2/3

(MD029, ol-prefix)